placeSpecificContent/hq/chatterboxes/chatterbox/buttons/privacyButton_client.js

```diff
@@ -0,0 +1,37 @@
+//
+//  privacyButton_client.js
+//
+//  Created by Rebecca Stankus on 0/09/19.
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() {
+
+    var _this;
+
+    var MAX_DISTANCE_TO_OPERATE_M = 3.11309;
```

Comment:
how'd you come up with this number? :D

Reply:
Alan and I chose a position to measure from.